src/main/kotlin/racingcar/model/CarManager.kt

```diff
@@ -0,0 +1,39 @@
+package racingcar.model
+
+import racingcar.entity.Car
+import racingcar.entity.Name
+import racingcar.misc.Util
+import racingcar.misc.Values
+
+class CarManager() {
+    private lateinit var cars: MutableList<Car>
+    fun init(names: List<Name>) {
+        cars = mutableListOf()
+        for (name in names) {
+            cars.add(Car(name))
+        }
```

Comment:
코틀린 컬렉션에는 다양한 api가 있어요! 
이경우에는 map을 사용하시면 좋을거같아요! 
```suggestion
        cars = names.map{ Car(it) }
```

Reply:
대부분의 for문을 map, mapIndexed 함수를 사용하여 변경하였습니다.